<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgba(0,0,0,.12);
border-radius: 7px;
background-color: white;</string>
     </property>
     <layout class="QGridLayout" name="gl_all">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gl_bar">
        <property name="leftMargin">
         <number>7</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>13</number>
        </property>
        <property name="horizontalSpacing">
         <number>7</number>
        </property>
        <item row="0" column="3">
         <widget class="QPushButton" name="pb_save_dataset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT Condensed</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../GUIImg/pb-save.png</normaloff>../../GUIImg/pb-save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLineEdit" name="le_overlap_ratio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid white;
	border-radius: 10px;
	background-color: #F1F1F1;
	padding-bottom: 1px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Overlap Ratio</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QPushButton" name="pb_clear_crop">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT Condensed</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #4B7CE1;
	color: #F7F9FF;
	border: 2px solid white;
	border-radius: 10px;
	padding-bottom: 1px;
}
QPushButton:pressed {
	margin: 1 1 1 1;
}</string>
          </property>
          <property name="text">
           <string>CLEAR</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLineEdit" name="le_crop_size">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid white;
	border-radius: 10px;
	background-color: #F1F1F1;
	padding-bottom: 1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Split Size</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pb_leave">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT Condensed</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../GUIImg/exit2.png</normaloff>../../GUIImg/exit2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLabel" name="info_display">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #EDF3FF;
border: 2px solid white;
border-radius: 10px;
padding-bottom: 1px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pb_openfile">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT Condensed</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../GUIImg/insert-picture-icon.png</normaloff>../../GUIImg/insert-picture-icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="le_crop_size_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid white;
	border-radius: 10px;
	background-color: #F1F1F1;
	padding-bottom: 1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pb_loadcsv">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT Condensed</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../GUIImg/placeholder.png</normaloff>../../GUIImg/placeholder.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="10">
         <widget class="QPushButton" name="pb_save_csv">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tw Cen MT Condensed</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #4379E3;
	color: #F7F9FF;
	border: 2px solid white;
	border-radius: 10px;
	padding-bottom: 1px;
}
QPushButton:pressed {
	margin: 1 1 1 1;
}</string>
          </property>
          <property name="text">
           <string>SAVE CSV</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gl_mode">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pb_select_mode">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>44</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>44</width>
              <height>44</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9ABAFA;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../GUIImg/cursor.png</normaloff>../../GUIImg/cursor.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pb_crop_mode">
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>44</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>44</width>
              <height>44</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border: None;
	border-radius: 5px;
}

QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}

QPushButton:hover {
	background-color: #EDF3FF;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../GUIImg/bnbox.png</normaloff>../../GUIImg/bnbox.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>100</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gl_canvas">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
